Changelog for v3.5.0 (#110)
MapsIndoors/mapsindoors.github.io: collections/_android/v3/CHANGELOG.md

### Security     in case of vulnerabilities.
-->

## [3.5.0] - 2020-06-16

### Added

> - Selection of default Floor when panning between buildings

Reviewer: ```suggestion
- Selection of default Floor when panning between Buildings
```

> ### Fixed
> - POIs disappear after applying Category Filter
> - Fixed an issue in the FloorSelector that prevented a floor selector to hide in some cases

Reviewer: ```suggestion
### Fixed

- Locations disappear after applying Category Filter
- Fixed an issue in the Floor Selector that prevented it from hiding in some cases
```
